portal-backend/depmap/data_page/api.py

```diff
@@ -179,3 +181,40 @@ def get(self):
         all_data_dict = _format_data_availability_summary_dict(formatted_df)
 
         return all_data_dict
+
+
+def _format_lineage_availability_summary_dict(summary_df: pd.DataFrame, data_type: str):
+    models_available = (
+        summary_df.loc[data_type]
+        .explode()[summary_df.loc[data_type].explode() == True]
+        .index.tolist()
+    )
+
+    lineage_counts = DepmapModel.get_lineage_primary_disease_counts(models_available)
+
+    summary = {
+        "lineage_counts": lineage_counts,
+    }
```

Comment:
Originally, I had lineage_counts as part of a larger dictionary before separating the lineage_availability out into its own endpoint. The "lineage_counts" is carry over from that, but as you point out, I could change it if you think it would improve readability.